### ISSUE TITLE
Package jsonlint-lines-primitives has no license

### ISSUE THREAD

**Issue by kozikkamil** [mapbox/mapbox-gl-js#11331]
Hey all! I see that the package jsonlint-lines-primitives has no license in npm. It's an issue when using Snyk to make sure that there are no terms of service violations when including such dependencies into the project. Is there any change we could license it in npm?

**Comment by mourner** [comment]
Good catch! There's license information (MIT) in [the dependency's readme](https://github.com/mapbox/jsonlinthttps://github.com/mapbox/jsonlint), but we should add it as a LICENSE file and publish to NPM so that automated tools could pick it up.

**Comment by kozikkamil** [comment]
That link doesn't work :suspect:

**Comment by MCuello17** [comment]
Any updates regarding this?

**Comment by skydiver** [comment]
Hey @mourner, can we add the license file to the repo?
Thanks!

**Comment by vikbert** [comment]
Same trouble with license file.
We used the package `mapbox`, which depends on @mapbox/jsonlint-lines-primitives. But there is no licence file and info in `jsonlint-lines-primitives`.

Can we add the missing license file?
